lit_tests/coloring.swift

```diff
@@ -1,404 +0,0 @@
-
-// CHECK: <kw>func</kw> <placeholder><#test1#></placeholder> () {}
-func <#test1#> () {}
-
-func funcTakingFor(for internalName: Int) {}
-// CHECK: <kw>func</kw> <id>funcTakingFor</id>(<id>for</id> <id>internalName</id>: <type>Int</type>) {}
-
-func funcTakingIn(in internalName: Int) {}
-// CHECK: <kw>func</kw> <id>funcTakingIn</id>(<id>in</id> <id>internalName</id>: <type>Int</type>) {}
-
-_ = 123
-// CHECK: <int>123</int>
-_ = -123
-// CHECK: <op>-</op><int>123</int>
-_ = -1
-// CHECK: <op>-</op><int>1</int>
-_ = -0x123
-// CHECK: <op>-</op><int>0x123</int>
-_ = -3.1e-5
-// CHECK: <op>-</op><float>3.1e-5</float>
-
-"--\"\(x) --"
-// CHECK: <str>"--\"</str>\<anchor>(</anchor><id>x</id><anchor>)</anchor><str> --"</str>
-
-func keywordAsLabel1(in: Int) {}
-// CHECK: <kw>func</kw> <id>keywordAsLabel1</id>(<id>in</id>: <type>Int</type>) {}
-func keywordAsLabel2(for: Int) {}
-// CHECK: <kw>func</kw> <id>keywordAsLabel2</id>(<id>for</id>: <type>Int</type>) {}
-func keywordAsLabel3(if: Int, for: Int) {}
-// CHECK: <kw>func</kw> <id>keywordAsLabel3</id>(<id>if</id>: <type>Int</type>, <id>for</id>: <type>Int</type>) {}
-func keywordAsLabel4(_: Int) {}
-// CHECK: <kw>func</kw> <id>keywordAsLabel4</id>(<kw>_</kw>: <type>Int</type>) {}
-func keywordAsLabel5(_: Int, for: Int) {}
-// CHECK: <kw>func</kw> <id>keywordAsLabel5</id>(<kw>_</kw>: <type>Int</type>, <id>for</id>: <type>Int</type>) {}
-func keywordAsLabel6(if func: Int) {}
-// CHECK: <kw>func</kw> <id>keywordAsLabel6</id>(<id>if</id> <id>func</id>: <type>Int</type>) {}
-
-func foo1() {
-// CHECK: <kw>func</kw> <id>foo1</id>() {
-  keywordAsLabel1(in: 1)
-// CHECK: <id>keywordAsLabel1</id>(<id>in</id>: <int>1</int>)
-  keywordAsLabel2(for: 1)
-// CHECK: <id>keywordAsLabel2</id>(<id>for</id>: <int>1</int>)
-  keywordAsLabel3(if: 1, for: 2)
-// CHECK: <id>keywordAsLabel3</id>(<id>if</id>: <int>1</int>, <id>for</id>: <int>2</int>)
-  keywordAsLabel5(1, for: 2)
-// CHECK: <id>keywordAsLabel5</id>(<int>1</int>, <id>for</id>: <int>2</int>)
-
-  _ = (if: 0, for: 2)
-// CHECK: <kw>_</kw> = (<id>if</id>: <int>0</int>, <id>for</id>: <int>2</int>)
-  _ = (_: 0, _: 2)
-// CHECK: <kw>_</kw> = (<kw>_</kw>: <int>0</int>, <kw>_</kw>: <int>2</int>)
-}
-
-func foo2(O1 : Int?, O2: Int?, O3: Int?) {
-  guard let _ = O1, var _ = O2, let _ = O3 else { }
-// CHECK:  <kw>guard</kw> <kw>let</kw> <kw>_</kw> = <id>O1</id>, <kw>var</kw> <kw>_</kw> = <id>O2</id>, <kw>let</kw> <kw>_</kw> = <id>O3</id> <kw>else</kw> { }
-  if let _ = O1, var _ = O2, let _ = O3 {}
-// CHECK: <kw>if</kw> <kw>let</kw> <kw>_</kw> = <id>O1</id>, <kw>var</kw> <kw>_</kw> = <id>O2</id>, <kw>let</kw> <kw>_</kw> = <id>O3</id> {}
-}
-
-func keywordInCaseAndLocalArgLabel(_ for: Int, for in: Int, class _: Int) {
-// CHECK:  <kw>func</kw> <id>keywordInCaseAndLocalArgLabel</id>(<kw>_</kw> <id>for</id>: <type>Int</type>, <id>for</id> <id>in</id>: <type>Int</type>, <id>class</id> <kw>_</kw>: <type>Int</type>) {
-  switch(`for`, `in`) {
```

Comment:
I think some test case with escaped identifiers would be good to keep.

Reply:
I think that's good👍. I added two in my new commit in https://github.com/apple/swift-syntax/pull/1953.